pkg/runit/servicebuilder.go

```diff
@@ -74,6 +74,7 @@ var DefaultBuilder = &ServiceBuilder{
 }
 
 var DefaultChpst = "/usr/bin/chpst"
+var DefaultContain = "/usr/bin/contain"
```

Comment:
This shouldn't be in the `runit` package. I'd move it to wherever `DefaultNolimit` is right now